models/intermediate/int_klaviyo__person_campaign_flow.sql

```diff
@@ -0,0 +1,78 @@
+with events as (
```

Comment:
this model seems overly-complicated.  How can we simplify?

- Do we need all of these run queries? What if we just have two lists (one for conversion metrics and one from revenue metrics) that have default values but the customer can edit if they want?